policyengine_us/parameters/gov/states/oh/tax/income/credits/exemption/amount.yaml

```diff
@@ -0,0 +1,27 @@
+description: Ohio limits its personal exemption, depending on modified adjusted gross income for the taxable year. 
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 2_350
+  - threshold: 
+      2021-01-01: 40_001
+    amount:
+      2021-01-01: 2_100
+  - threshold:
+      2021-01-01: 80_001
+    amount:
+      2021-01-01: 1_850
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  period: year
+  label: Ohio Personal Exemption
+  reference:
+    - title: Ohio Revised Code - Section 5747.025： Personal exemptions (A)
+      href: https://codes.ohio.gov/ohio-revised-code/section-5747.025
+    - title: 2022 Ohio IT 1040 Individual Income Tax Return Form
```

Comment:
add pages to all 2022 and 2021 tax form references

Reply:
page where the number can be found

Reply:
> 
I think 2021 and 2022 both refer to Ohio Revised Code - Section 5747.025： Personal exemptions (A)(1)(2)(3). Does it mean we need to write as follows:

` - threshold:
      2021-01-01: 0
    amount:
      2021-01-01: 2_350
  - threshold: 
      2021-01-01: 40_001
    amount:
      2021-01-01: 2_100
  - threshold:
      2021-01-01: 80_001
    amount:
      2021-01-01: 1_850
 - threshold:
      2022-01-01: 0
    amount:
      2022-01-01: 2_350
  - threshold: 
      2022-01-01: 40_001
    amount:
      2022-01-01: 2_100
  - threshold:
      2022-01-01: 80_001
    amount:
      2022-01-01: 1_850`